x-pack/plugins/spaces/public/management/edit_space/enabled_features/feature_table.tsx

```diff
@@ -101,6 +101,7 @@ export class FeatureTable extends Component<Props, {}> {
 
         return (
           <EuiSwitch
+            data-test-subj={`${record.feature.id}-toggle`}
```

Comment:
Nit
```suggestion
            data-test-subj={`feature-${record.feature.id}-toggle`}
```

Reply:
Thanks for noticing the typo Joe.

Reply:
And the data-test-subj suggestion. Neat.